packages/rocketchat-setup-wizard/server/lib/getWizardSettings.js

```diff
@@ -0,0 +1,15 @@
+Meteor.methods({
+	getWizardSettings(user) {
+		if (!user && !Meteor.userId()) {
+			return false;
+		}
+
+		user = Meteor.userId();
+
+		if (RocketChat.authz.hasRole(user, 'admin') && RocketChat.models && RocketChat.models.Settings) {
+			return RocketChat.models.Settings.find({wizard: {'$exists': true, '$ne': null}}).fetch();
```

Comment:
Can you add the query to the model file and call the method here?
Is necessary to have the `'$ne': null`?

Reply:
Yes I'll add to the model files.
The `'$ne': null` is necessary because all settings that doesn't have the `wizard` object will return `wizard: null`.